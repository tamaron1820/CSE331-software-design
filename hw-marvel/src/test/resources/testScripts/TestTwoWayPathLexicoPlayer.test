# Tests that if team(In Marvel's example the book) have many children
# priority increases in lexicographical order.

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/soccerPlayer.csv

LoadGraph soccer soccerPlayer.csv

FindPath soccer Mbappe Pedri

FindPath soccer Pedri Mbappe