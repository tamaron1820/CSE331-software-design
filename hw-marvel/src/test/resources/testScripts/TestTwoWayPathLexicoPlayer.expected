# Tests that if team(In Marvel's example the book) have many children
# priority increases in lexicographical order.

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/soccerPlayer.csv

loaded graph soccer

path from Mbappe to Pedri:
Mbappe to Messi via Paris-Saint-Germain
Messi to Pedri via Barcelona

path from Pedri to Mbappe:
Pedri to Messi via Barcelona
Messi to Mbappe via Paris-Saint-Germain